### PR TITLE
Remove DNX

### PR DESCRIPTION
**Pull request by danroth27**
Remove DNX related files and references.

### CODE REVIEW

**Comment by danroth27** [comment]
@shirhatti @Rick-Anderson @rachelappel @blackdwarf @glennc  Please review

**Comment by shirhatti** [comment]
A List of items that still need updating
- [ ] [Servicing](https://github.com/aspnet/Docs/blame/master/aspnet/hosting/servicing.rst#L18)
- [ ] Secret manager- Wherever ``user-secret`` is referenced, it is usually prefaced with ``dnu commands install ...``
    1. [Facebook auth](https://github.com/aspnet/Docs/blob/master/aspnet/security/authentication/sociallogins.rst#use-secretmanager-to-store-facebook-appid-and-appsecret)
    2. [Secrets Manager](https://github.com/aspnet/Docs/blob/master/aspnet/security/app-secrets.rst#installing-the-secret-manager-tool)

**Comment by danroth27** [comment]
One more:
- [ ] [Web API migration final sample](https://docs.asp.net/en/latest/tutorials/first-web-api.html)

File: aspnet/dnx/commands.rst [hunk 1]
```diff
@@ -1,235 +0,0 @@
-Using Commands
```
Comment (shirhatti) [inline aspnet/dnx/commands.rst]: I'm guessing documentation on `dotnet tools` will live in the core-docs?

File: aspnet/publishing/web-publishing-vs.rst [hunk 2]
```diff
@@ -21,11 +21,11 @@ The following image shows a visualization of this process.
 
 In the image above each black circle ● indicates an extension point, we will cover each extension point later in this document.
 
-When you start a publish operation, the publish dialog is closed and then MSBuild is called to start the process. Visual Studio calls MSBuild to do this so that you can have parity with publishing when using Visual Studio or the command line. The MSBuild layer is pretty thin, for the most part it just calls dnu publish. Let’s take a closer look at dnu publish.
+When you start a publish operation, the publish dialog is closed and then MSBuild is called to start the process. Visual Studio calls MSBuild to do this so that you can have parity with publishing when using Visual Studio or the command line. The MSBuild layer is pretty thin, for the most part it just calls ``dotnet publish``. Let’s take a closer look at ``dotnet publish``.
```
Comment (shirhatti) [inline aspnet/publishing/web-publishing-vs.rst]: We'll have to update this article to use the new `dotnet publish-iis` gesture.

File: aspnet/security/authentication/sociallogins.rst [hunk 3]
```diff
@@ -85,8 +85,7 @@ The project created has code in Startup which reads the configuration values fro
 
 Follow these steps to add the Facebook AppId and AppSecret to the Secret Manager:
 
-- Use DNVM (.NET Version Manager) to set a runtime version by running **dnvm use 1.0.0-beta8**
-- Install the SecretManager tool using DNU (Microsoft .NET Development Utility) by running **dnu commands install Microsoft.Extensions.SecretManager**
+- Install the SecretManager tool by running **dnu commands install Microsoft.Extensions.SecretManager**
```
Comment (shirhatti) [inline aspnet/security/authentication/sociallogins.rst]: Replace with `dotnet` tools

File: aspnet/dnx/commands.rst (hunk 1 above)
Comment (danroth27) [inline aspnet/dnx/commands.rst]: Yup.

File: aspnet/publishing/web-publishing-vs.rst (hunk 2 above)
Comment (danroth27) [inline aspnet/publishing/web-publishing-vs.rst]: We augment "dotnet publish" instead of creating a separate "publish-iis" command that the user calls.

**Comment by shirhatti** [comment]
:shipit:

**Comment by danroth27** [comment]
Filed https://github.com/aspnet/Docs/issues/1127 to track the yeoman screenshot updates while we wait for updated generators.

**Pull request merged**